package.json

```diff
@@ -165,11 +170,13 @@
     "cozy-pouch-link": "5.7.6",
     "cozy-stack-client": "5.6.1",
     "cozy-ui": "19.3.0",
+    "csv-writer": "^1.2.0",
```

Comment:
can you fix version? (remove ^ and execute `yarn`)

Reply:
Si, j'en ai besoin dans `service-fixture.spec.js` ligne 63

Reply:
Je n'avais pas vu ton commentaire pour `devDependencies, je change

Reply:
Suppression de `^`